docker-build.sh

```diff
@@ -5,6 +5,123 @@
 set -o errexit
 set -o xtrace
 
+ARCHIVE_ADDR=https://archive.ubuntu.com/ubuntu/
+PORTS_ADDR=https://ports.ubuntu.com/
+
+# Prepare HWA headers, libs and drivers for x86_64-linux-gnu
+prepare_hwa_amd64() {
+    # Download and install the nvidia headers from deb-multimedia
+    pushd ${SOURCE_DIR}
+    git clone --depth=1 https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+    pushd nv-codec-headers
+    make
+    make install
+    popd
+    popd
+
+    # Download and setup AMD AMF headers from AMD official github repo
+    # https://www.ffmpeg.org/general.html#AMD-AMF_002fVCE
+    svn checkout https://github.com/GPUOpen-LibrariesAndSDKs/AMF/trunk/amf/public/include
+    pushd include
+    mkdir -p /usr/include/AMF && mv * /usr/include/AMF
+    popd
+
+    # Download and install libva
+    pushd ${SOURCE_DIR}
+    git clone -b v2.6-branch --depth=1 https://github.com/intel/libva
+    pushd libva
+    sed -i 's|getenv("LIBVA_DRIVERS_PATH")|"/usr/lib/jellyfin-ffmpeg/dri"|g' va/va.c
+    sed -i 's|getenv("LIBVA_DRIVER_NAME")|NULL|g' va/va.c
```

Comment:
why do we need these instead of just exporting those variables somewhere?..

Reply:
System variables are unstable and can be easily modified during system update or software package uninstallation. Will affect the use of the vaapi driver provided by the official repository of Ubuntu/Debian. And it will cause the `libva` included in the deb package to not find the integrated intel driver. My purpose is to make the dynamic link libraries integrated in the `jellyfin-ffmpeg` deb package coexist with those already existing in those systems. After all, some people still use linux desktop for daily use.

Reply:
I wasn't suggesting to export those globally. Also, I think I am now proposing a better solution: https://github.com/jellyfin/jellyfin-ffmpeg/pull/33#discussion_r406145592 (needs to be checked if it works, though).